src/test/regress/expected/multi_reference_table.out

```diff
@@ -1308,26 +1308,26 @@ SELECT logicalrelid FROM pg_dist_shard WHERE logicalrelid::regclass::text LIKE '
 (0 rows)
 
 -- now test DDL changes
-CREATE TABLE reference_table_ddl (value_1 int, value_2 float, value_3 text, value_4 timestamp);
-SELECT create_reference_table('reference_table_ddl');
+CREATE TABLE reference_schema.reference_table_ddl (value_1 int, value_2 float, value_3 text, value_4 timestamp);
+SELECT create_reference_table('reference_schema.reference_table_ddl');
  create_reference_table 
 ------------------------
  
 (1 row)
 
 -- CREATE & DROP index and check the workers
-CREATE INDEX reference_index_1 ON reference_table_ddl(value_1);
-CREATE INDEX reference_index_2 ON reference_table_ddl(value_2, value_3);
+CREATE INDEX reference_index_1 ON reference_schema.reference_table_ddl(value_1);
+CREATE INDEX reference_index_2 ON reference_schema.reference_table_ddl(value_2, value_3);
```

Comment:
couldn't we just use search_path here? that's the pattern followed by some other tests.

Reply:
Depends on what exactly we want to test. Is the test more interesting with using the search_path or without using it?

Reply:
Both are interesting. Anyway, no strong objections to keep it this way.